### DISCUSSION TITLE
Error code

### DISCUSSION THREAD

**Discussion by dnimigon** [flybywiresim/aircraft#7534] [Help]
i am installing the FS Live traffic Liveries and getting this error code. Not sure what it means
AKKQ9AZAISVZYT. 
Its when i tried to install the  A320-25 1N
A32NX

**Comment by ExampleWasTaken** [comment]
I recommend joining the Discord as there support will be able with your specific code.
You can join the Discord here: https://discord.gg/flybywire
Upon joining navigate to the #a32nx-support channel and describe your issue along with posting the code that you already provided above.
